odoo/_monkeypatches/PIL.py

```diff
@@ -0,0 +1,14 @@
+import PIL
+
+
+def patch_module():
+    try:
+        from PIL.Image import Palette, Resampling, Transpose  # noqa: F401, PLC0415
+    except ImportError:
+        PIL.Image.Transpose = PIL.Image
+        PIL.Image.Palette = PIL.Image
+        PIL.Image.Resampling = PIL.Image
```

Comment:
For the record, I don't like this.

Reply:
and remove the ugly `if monky not in sys.module: patch_module()`, I honestly don't care if PIL is not preloaded if someone explicitly import this tools.

Reply:
I agree that we could monkeypatch directly in `tools/images.py`. It is a compatiblity issue, that we could solve in tools and avoid monkeypatching always PIL.

Reply:
I will remove the commit, no problem.

Though I find that having compatibility issue solving and monkeypatching always in one place, is just like having all tea mugs in the kitchen: 
- know where to find them
- see them all on the counter
- don't find old mugs in the sideboard 
- when you buy another one it already has a place

So please don't vomit on my proposals. I also find that the use case of importing `odoo.tools.image` without `odoo.monekypatches` is kinda exception thinking. 

If we want, we can modify `odoo.monkeypatch.patch_init(module=None)` to allow passing a name of a module you want monkeypatched, `import odoo.monkeypatch.patch_init` here and do like `PIL = patch_init('PIL')`. Would that look any better to you?

Reply:
The decision to either move this to monkeypatches or not was not very clear.
Thank you for you work.

Reply:
Borderline emojis apart - The fact that you both are considering and integrating, or reworking part of my work, making it even better, to me it's not a win, it's a **triumph**